indicators/NO_BIRD_001-004/R/NO_BIRD_001-004.qmd

```diff
@@ -0,0 +1,624 @@
+
+## 4. Reference condition and levels
+
+### 4.1 Reference condition
+
+In Norway, the reference condition for natural ecosystems have been one of largely intact ecosystems without large-scale human modifications [@nybø2017]. For semi-natural ecosystems, human influence is allowed, and indeed a prerequisite, but then in the sense of appropriate, stable and traditional land-use practices. Fro agricultural ecosystems we lack a precise definition for a reference condition. The metrics are not normalised, and therefore there is no direct way to compare the values to a reference condition.
+
+### 4.2 Reference levels {#sec-rl}
+
+None. Note that the use of a baseline year to center the trend analyses is not equivalent to the use of reference levels, and do not allow quantitative statement of the type *this value is very high/low.* Note also that reference levels for individual bird species has been set in the Norwegian Nature Index, and it is possible to compile similar normalised bird indices by combining these single bird indicators [see @nybø2023 page 55] .
+
+## 5. Uncertainties
+
+One source of uncertainty comes from the assumption that multi-species abundance indices use a representative selection of species sampled in a representative set of habitats. However, due to resource- and logistic constraints, that is often not the case and non-random sampling along, for example, rural-urban gradients may bias breeding bird indices derived from data from "hekkefuglovervåkingen" [@husby2020].
+
+## 6. References
+
+::: {#refs}
+:::
+
+## 7. Datasets
+
+### 7.1 Dataset A - Norsk Hekkefulgovervåking {#sec-data1}
+
+Data is downloaded from the [internal NINA database](https://hekkefuglovervakingen.nina.no/Fugl/Default.aspx?ReturnUrl=/Fugl/). See @sec-aboutdata for details on the dataset.
+
+## 8. Spatial units
+
+The sampling locations are selected with a regional stratification, but we do not try to account for this spatial structure here (see @sec-aboutdata for discussion). Therefore the only spatial unit we rely on the the national scale (the ecosystem accounting area).
+
+## 9. Analyses
+
+### Workflow setup {#sec-workflow}
+
+The code for the following analyses has been developed in a separate GitHub repository: https://github.com/NINAnor/birdIndicators As a first step, we therefore need to read in all the functions from that repository. The repository has not been converted into a package yet, so we need to download the entire repository and source all function files in the “R” folder manually:
+
+```{r}
+#| eval: false
+#| code-summary: "Download functions from GitHub repository"
+temppath <- here::here("indicators/NO_BIRD_001-004/temp")
+destpath <- paste0(temppath, "/birdIndicators_Repo.zip")
+
```

Comment:
It's unclear to me how I can get the birdIndicator_Repo.zip file that includes the functions to run the code.

Reply:
I hashed out the line that downloads the repo (line 246). Perhaps I should rather put it in an `eval: false` block?

Reply:
This has been fixed I think by adding the `if(!file.exists(...` part below.